TarSCM/scm/base.py

```diff
@@ -142,6 +148,27 @@ def _calc_repocachedir(self):
             self.repohash = self.get_repocache_hash(self.args.subdir)
             self.repocachedir = os.path.join(repocachedir, self.repohash)
 
+    def _calc_proxies(self):
+        # check for standard http/https proxy variables
+        #   - http_proxy
+        #   - https_proxy
+        #   - no_proxy
+        httpproxy  = os.getenv('http_proxy')
+        httpsproxy  = os.getenv('https_proxy')
+        noproxy  = os.getenv('no_proxy')
+
```

Comment:
I`m a bit confused about upper/lower case of these variables.
In "/etc/sysconfig/proxy" upper case variables are used.
In "/etc/profile.d/profile.sh" lower case variables are set from the upper case variables
In "/etc/wgetrc" lower case is used 
The curl manpage says:

```
The environment variables can be specified in lower case or upper case. 
The lower case version has precedence. 
http_proxy is an exception as it is only available in lower case.
```

Reply:
Actually /etc/profile.d/profile.sh reads HTTP[S]_PROXY (uppercase) from /etc/sysconfig/proxy and declares http[s]_proxy (lowercase) on the system. The sysconfig file is YaST-specific. Wget  respects the lower case variant, as curl does.